linux_sql/psql_docker.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+if [ $# != 2 ];
+then
+	echo $#
+	echo "Wrong number of arugments"
+	exit 1
+fi
+if [ $1 == "start" ]; then 
+	systemctl status docker  || systemctl start docker
+	linecount=`docker ps -af name=jrvs-psql | wc -l`
+	if [ "$linecount" -gt 2 ];
+	then
+		docker run --name jrvs-psql -e POSTGRES_PASSWORD=$PGPASSWORD -d -v pgdata:/var/lib/postgresql/data -p 5432:5432 postgres
+	fi
```

Comment:
if linecount is > 2, then more than 1 container with name jrvs-psql exist, thus no extra container required.